po/ja.po

```diff
@@ -18290,25 +18290,35 @@ msgid ""
 "Rust uses the type system to enforce synchronization of shared data. This is "
 "primarily done via two types:"
 msgstr ""
+"Rustは共有されたデータの同期を強制するために型システムを利用します。これは主"
+"に2つの型により行われます："
 
 #: src/concurrency/shared_state.md:6
 msgid ""
 "[`Arc<T>`](https://doc.rust-lang.org/std/sync/struct.Arc.html), atomic "
 "reference counted `T`: handles sharing between threads and takes care to "
 "deallocate `T` when the last reference is dropped,"
 msgstr ""
+"[`Arc<T>`](https://doc.rust-lang.org/std/sync/struct.Arc.html), アトミックに"
+"参照カウントする型 （atomic reference counted）を `T` とするとき : スレッド間"
```

Comment:
Translation confidence: Low, Re: "atomic reference counted"

Reply:
Hmm, this is a hard one. This basically means that T has a reference count and its count is managed atomically, right?

I would probably just leave it un-translated up to ':".

Reply:
Yes, that is my understanding. I agree, the reader only need to learn that Arc stands for atomic reference counted, rather than get confused with the translation?

FYI, the Rustbook JP translated as "原子的に参照する型." [状態共有並行性 - Rustbook](https://doc.rust-jp.rs/book-ja/ch16-03-shared-state.html#arct%E3%81%A7%E5%8E%9F%E5%AD%90%E7%9A%84%E3%81%AA%E5%8F%82%E7%85%A7%E3%82%AB%E3%82%A6%E3%83%B3%E3%83%88) but this sounds confusing. At least, "原子的"->"アトミック" is better in the programming context, IMO.